AUTHORS.txt

```diff
@@ -366,3 +337,6 @@ Simon Legner <Simon.Legner@gmail.com>
 Vladimir Sitnikov <sitnikov.vladimir@gmail.com>
 Anders Kaseorg <andersk@mit.edu>
 Alex <aleksandrosansan@gmail.com>
+Timo Tijhof <krinkle@fastmail.com>
+Gabriela Gutierrez <gabigutierrez@google.com>
+Dimitri Papadopoulos Orfanos <3234522+DimitriPapadopoulos@users.noreply.github.com>
```

Comment:
I know the number of changes looks odd, but I've confirmed that `grunt authors` has the same result. My guess is this has something to do with privacy rules being respected more accurately in git logs.

Reply:
No, and I actually made these changes long ago on purpose... This was to certify that contributions to Sizzle matter as much to jQuery as contributions to the main jQuery repo. I changed the file so that the entries are ordered by the first contribution to any of those two repositories. I'd like to preserve that.

Cf. https://github.com/jquery/jquery/pull/4395/files#diff-b897336dda0579c40f3623b68eb6652ded79e206f4cf1af7ceb24b683fd46ff1 & https://github.com/jquery/jquery/pull/5113/files#diff-b897336dda0579c40f3623b68eb6652ded79e206f4cf1af7ceb24b683fd46ff1

Reply:
I see. Let's incorporate that into the authors script then.

Reply:
I've updated the authors script to clone the sizzle repo, combine the authors, sort by their first contribution, ensure they are unique by name, and then run cleanup so the sizzle repo doesn't stick around.

One case that I found interesting is Timo's entry. I noticed that his old email is still in the authors, whereas both emails were in there before. I think we only need the one and the email will automatically update when Timo updates the mailmap.